docs/reference/Data/ClientCallback.md

```diff
@@ -0,0 +1,21 @@
+##### Data Type : Security
+##### ClientCallback - ClientCallback  data structure.
+---
+```
+#include <bsafe.h>
+```
+
+**Definition :**
+```
+typedef BOOL (*ClientCallback) (char *pAzp, void *vpCallerContext); 
```

Comment:
Is there detailed description about the clientcallback type.

Reply:
No